controllers/networking-calico/pkg/apis/calico/doc.go

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +k8s:deepcopy-gen=package
+// +groupName="calico.networking.extensions.gardener.cloud"
```

Comment:
```suggestion
// +groupName="calico.network.extensions.gardener.cloud"
```

Reply:
I used networking instead allover because thats also how the packages are named and all the other resources.

Reply:
The extension resource is called `Network`. Please follow a consistent approach. Either rename the extension resource or rename these implementations.

Reply:
What about this one? I see you are still using `networking-calico`, so do you plan to rename `Network` CRD to `Networking`? Or vice versa?

Reply:
yes I chose networking over network, since everything was already networking. Changing to network would be some unneeded overhead in my opinion. If you really want the CRD to change to Networking, can do that as well?